Vulkan-Pipeline-Assistant/mainwindow.h

```diff
@@ -55,14 +57,16 @@ namespace vpa {
         QWidget* MakeMultisampleBlock();
         QWidget* MakeDepthStencilBlock();
         QWidget* MakeRenderPassBlock();
+        void MakeDescriptorBlock();
 
-        QComboBox* MakeComboBox(QWidget* parent, QVector<QString> items);
+        void VulkanCreationCallback();
 
         Ui::MainWindow* m_ui;
         VulkanMain* m_vulkan;
```

Comment:
Add a note to trello to explicitly assign all empty data values in constructor.

Reply:
Complete